xocto/storage/s3_select.py

```diff
@@ -165,3 +170,25 @@ def get_serializers_for_csv_file(
         temp_dict["ScanRange"] = scan_range.to_dict()
 
     return temp_dict
+
+
+def get_serializers_for_parquet_file(
+    output_serializer: JSONOutputSerializer | CSVOutputSerializer,
+) -> dict[str, dict[str, Any]]:
+    """
+    Returns input and output serialization dictionaries that should be used to perform a select_object_content query.
+
+    Parquet files do not support input serialization or scan ranges.
+    """
+    temp_dict: dict[str, dict[str, Any]] = {
```

Comment:
You could just return this directly, without giving it this temp name

```suggestion
    return {
```